hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupInfoManagerImpl.java

```diff
@@ -592,7 +594,13 @@ private synchronized void refresh(boolean forceOnline) throws IOException {
 
     // This is added to the last of the list so it overwrites the 'default' rsgroup loaded
     // from region group table or zk
-    groupList.add(new RSGroupInfo(RSGroupInfo.DEFAULT_GROUP, getDefaultServers(groupList)));
```

Comment:
Why change this? getDefaultServers not used any more?

Reply:
`getDefaultServers` is still used in behind. This is just for version compatibility, not sure whether groupList has a default group or not after this patch.